qemu/tests/driver_load.py

```diff
@@ -94,18 +102,27 @@ def get_driver_id(cmd, pattern):
         session, params["driver_unload_cmd"])
     session.close()
 
-    for repeat in range(0, int(params.get("repeats", 1))):
-        error_context.context("Unload and load the driver. Round %s" % repeat,
-                              logging.info)
-        logging.info("Get driver info from guest")
-        driver_id = get_driver_id(driver_id_cmd, driver_id_pattern)
-
-        error_context.context("Unload the driver", logging.info)
-        unload_driver(driver_unload_cmd, driver_id)
-        time.sleep(5)
-        error_context.context("Load the driver", logging.info)
-        load_driver(driver_load_cmd, driver_id)
-        time.sleep(5)
+    if stop_service_cmd:
+        logging.info("Stop service before driver load testing")
```

Comment:
Had better to specify which service in the log.

Reply:
we can get it from the cmd output